test.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import unittest
+
+
+class TestServerFunctionality(unittest.TestCase):
+
+	def setUp(self):
+		self.url= 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'
+		self.loginval= {"login":"test", "password":"test"}
+
+	def test_1Login(self):
+		requestLogin = requests.post(self.url+'/login', json= self.loginval)
+		Token=(requestLogin.json().get('data')).get('tokenKey')
```

Comment:
1. variables should start from lover case character should be token 
2. use space between variable and expression token = (req...